src/css/controls/imports/controlbar.less

```diff
@@ -43,7 +43,7 @@
                 }
             }
 
-            .jw-icon-inline,
+            .jw-icon-inline:not(.jw-text-live),
```

Comment:
Approving, but idk if we should've used the jw-icon-line class for something that isn't an icon in the first place.